Resources/Prototypes/_Harmony/Catalog/spellbook_catalog.yml

```
@@ -1,4 +1,16 @@
 # Offensive
+- type: listing
+  id: SpellbookLesserGun
+  name: spellbook-lesser-gun-name
+  description: spellbook-lesser-gun-desc
+  productAction: ActionLesserSummonGun
+    cost:
```

This isn't indented properly, that's why your tests are failing


```suggestion
  cost:
```